os_site/src/index.njk

```diff
@@ -0,0 +1,36 @@
+---
+layout: base.njk
+---
+
+{% from "macros/post-card.njk" import postCard %} 
+{% from "macros/project-card.njk" import projectCard %} 
+
+<main>
+    <section aria-label="open source projects">
```

Comment:
Nice use of the section tag. You have an h2 down below so might even be able to drop the `aria-label` attribute. I'm thinking of [method 2 here](https://css-tricks.com/how-to-section-your-html/#method-2-add-a-h-element-to-it)

Reply:
Nice! Thanks for linking that. Just removed those labels.